include/tvm/relay/function.h

```diff
@@ -144,6 +144,9 @@ constexpr const char* kComposite = "Composite";
 constexpr const char* kInline = "Inline";
 /*! \brief Indicate the function was created by the Pattern Partitioning Pass. */
 constexpr const char* kPartitionedFromPattern = "PartitionedFromPattern";
+
+/*! \brief Mark the function as only composed of reshape operations. */
+constexpr const char* kReshapeOnly = "relay.reshape_only";
```

Comment:
are there other ops that could benefit? for example `squeeze`, `expand_dims`, etc.? I feel like this should more generally be like `relay.alias_only` (although alias might be too general since you could "alias" a subset of an input by slicing)